src/webview/webkitgtk/mod.rs

```diff
@@ -390,6 +399,40 @@ impl InnerWebView {
   pub fn zoom(&self, scale_factor: f64) {
     WebViewExt::set_zoom_level(&*self.webview, scale_factor);
   }
+
+  pub fn find_in_page<F>(&self, string: String, option: FindInPageOption, f: F)
+  where
+    F: Fn(bool) + 'static,
+  {
+    struct WithClosure {
+      f: Box<dyn Fn(bool) + 'static>,
+    }
+
+    let mut flags = FindOptions::NONE;
+    if option.backwards {
+      flags |= FindOptions::BACKWARDS;
+    }
+    if !option.case_sensitive {
+      flags |= FindOptions::CASE_INSENSITIVE;
+    }
+    if option.wraps {
+      flags |= FindOptions::WRAP_AROUND;
+    }
+
+    self
+      .find_controller
+      .search(&string, flags.bits(), option.max_match_count);
+
+    let handler = Rc::new(WithClosure { f: Box::new(f) });
```

Comment:
Why wrap in a struct? can't you just do?
```suggestion
    let handler = Rc::new(Box::new(f));
```

Reply:
Thank you for your review.
I fixed your review.